### ISSUE TITLE
Emulator freezing on loading a savestate

### ISSUE THREAD

**Issue by focamillo** [melonDS-emu/melonDS#1884]
Happens on Steam Deck. Before loading a ROM I can access the emu's UI, the settings etc.
On loading, the game (Pokemon) still runs, is playable, but as soon as I interact with the emulator, eg loading a savestate or changing settings the Emulator freezes completely. (Not shure if the latest Steam Deck system update caused any issues with that, didn't happen before...)

**Comment by 10hourslong** [comment]
I'm actually experiencing a similar issue on my Deck. If you don't mind me asking, does the application hard crash when opening settings like "Input settings", "Emu settings", "Setup cheat codes", etc. only while the game is running? Because that started happening to me just a few days ago and I just can't seem to find the culprit of what's causing it after a lot of troubleshooting.

**Comment by focamillo** [comment]
Exactly, only while the game is running. Everything's fine before...

**Comment by PBRasmussen** [comment]
I have this same issue as well, is there any fix for this at all?

**Comment by JesseTG** [comment]
Which version of melonDS are you using? The most recent release from last year, or the current `master`?

**Comment by PBRasmussen** [comment]
This is happening with the version installed by emudeck, 0.9.5, not sure which specific release that is

**Comment by focamillo** [comment]
Happens with the one from the package manager (Discover) as well and also with the standalone version from the website.

**Comment by 10hourslong** [comment]
I've been trying the standalone version through EmulationStation and through the Discover flatpak. Both seem to be having this issue, I've tried getting a fresh new set of BIOs, doing a complete wipe/reinstall through both methods, removing the .ini file, but nothing I tried seemed to work. I was beginning to think something might have gotten corrupted deep in the files that I cannot find, but now that I know I'm not the only one having this issue, seems like it might be a a Deck compatibility issue. I haven't tested this on a normal PC yet, so I'm not sure if any similar issues currently exist on Windows.
